package.json

```diff
@@ -122,7 +122,7 @@
   },
   "prettier": "@prisma-labs/prettier-config",
   "peerDependencies": {
-    "@prisma/client": "^3.5",
```

Comment:
This is a breaking change and should not be done without explanation. For example what is the rationale given for this going to be in the changelog?